src/main/kotlin/com/team4099/robot2023/subsystems/drivetrain/swervemodule/SwerveModuleIOTalon.kt

```diff
@@ -227,21 +226,21 @@ class SwerveModuleIOTalon(
     )
     Logger.recordOutput("$label/drivePositionUnits", driveSensor.position.inMeters)
     inputs.drivePosition = driveSensor.position
-    inputs.steeringPosition = steeringSensor.position
+    inputs.steerPosition = steeringSensor.position
     Logger.recordOutput("$label/rawSteeringValue", steeringFalcon.position.value)
     Logger.recordOutput("$label/rawSteeringValue", steeringFalcon.position.value)
```

Comment:
remove

Reply:
I think in one of the controls meeting they refactored the name steering to be "steer"

Reply:
removed extra log